src/Models/FormSchema.cs

```diff
@@ -27,6 +27,10 @@ public class FormSchema
 
         public string GeneratedReferenceNumberMapping { get; set; }
 
+        public bool SavePaymentAmount { get; set; }
+
+        public string PaymentAmountMapping { get; set; } = "paymentAmount";
```

Comment:
Do we use this is hardcoded string more than one place if so maybe it could be a constant?